tests/test_trajectory.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+import unittest
+from ddt import ddt, data, unpack
+
+import numpy as np
+
+import broni
+from broni.shapes.primitives import Cuboid, Sphere
+
+
+@ddt
+class TestTrajectory(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def test_invalid(self):
```

Comment:
not obvious why these should be invalid when reading the test.

Reply:
changed the same of the method, not sure it will help much. more comments?